posthog/tasks/test/test_exporter.py

```diff
@@ -38,12 +39,12 @@ def test_exporter_runs(self, mock_get_driver: MagicMock, mock_uuid: MagicMock) -
         assert self.exported_asset.content is None
         assert self.exported_asset.content_location is not None
 
-    def test_exporter_setsup_selenium(self, mock_uuid: MagicMock) -> None:
```

Comment:
I think you could better use pytest.skip() here to mark it as a skipped test rather than commenting out

Reply:
![spiderhogs-54f1bf6e8ebf31791e4c5c1813b19f2a](https://user-images.githubusercontent.com/984817/189087884-7698ffce-afc2-46ef-a25a-1a30ff669698.png)